jetstream/core/metrics/prometheus.py

```diff
@@ -255,3 +255,12 @@ def get_request_output_length(self):
 
   def get_request_success_count_metric(self):
     return self._request_success_count.labels(id=self._id)
+
+  _total_tokens_in_current_batch = Gauge(
+      name="jetstream_total_tokens_in_current_batch",
```

Comment:
We have padding in each batch, I feel this metric is not critical or useful. We more care about total response tokens per request. @JoeZijunZhou Please share your thoughts on this metrics.

Reply:
Following up in the internal doc. The GKE team added this metric in the design doc.